### PR TITLE
Fix deepcopy discovery

### PR DESCRIPTION
**Pull request by rajatpratapbisht**
# obj: run mana on discovery cluster
1.  `Added Datatype for mana_handle union memeber for MPI_FILE`: added datatype signature  for vanilla MPI lib..
2. `Commented out Fortran f90 files from test Makefile`: fortran version incompatible on discovery cluster. All systems will not not have Fortran test files..
3. `fixed 16-bit alignment error on Discovery`: STACK_RLIMT.rlimit_cur is set to RLIMIT_INFIITY by default on discovery. This commit tests for this edge case and uses 32MB as rlimit_cur (soft limit) value. Without this commit test for 16-bit alignment fails for deepCopyStack()..

### CODE REVIEW

**Review by xuyao0127 — COMMENTED** [review]
There are some empty lines in this commit. Can you remove them?

File: mpi-proxy-split/lower-half/kernel-loader.cpp [hunk 1]
```diff
@@ -102,6 +102,8 @@ void set_addr_no_randomize(char *argv[]) {
 }
 
 int main(int argc, char *argv[], char *envp[]) {
+  
+
```
Comment (xuyao0127) [inline mpi-proxy-split/lower-half/kernel-loader.cpp:106]: There are some empty lines in this commit. Can you remove them?

Comment (rajatpratapbisht) [reply mpi-proxy-split/lower-half/kernel-loader.cpp]: Fixed it in the latest commit

**Review by xuyao0127 — APPROVED** [review]
LGTM
